### PR TITLE
add filter_tabular 3.3.0; preserve filter_tabular 2.0.0

### PR DESCRIPTION
**Pull request by eschen42**
@bgruening @hexylena 

This PR is to add `filter_tabular` to `tools_iuc.yaml`

Regarding the additions to `tools_iuc.yaml.lock`
`filter_tabular` 2.0.0 (revision 34d29339abab) is presently installed on UseGalaxy.eu
`filter_tabular` 3.3.0 (revision 4d5aae46f850) is to be installed onto UseGalaxy.eu

Thank you.

P.S. When I ran
```make fix```
it identified a number of issues that I did not include in this PR because it did not concern this tool.

### CODE REVIEW

File: tools_iuc.yaml.lock [hunk 1]
```diff
@@ -8362,6 +8362,12 @@ tools:
   revisions:
   - 549d2cb4c6f2
   tool_panel_section_label: Text Manipulation
+- name: filter_tabular
+  owner: iuc
+  revisions:
+  - 4d5aae46f850
```
Comment (bgruening) [inline tools_iuc.yaml.lock:8368]: do you need both version? can we just take the latest?

Comment (bgruening) [reply tools_iuc.yaml.lock:8368]: ping @eschen42

**Comment by eschen42** [comment]
@bgruening I removed the filter_tabular 2.0.0 version (which is presently installed on usegalaxy.eu) and left only the 3.3.0 version (which I need), assuming that this won't uninstall the old version and break others' workflows.  I also added v0.1.18 of galaxyp mqppep_anova and mqppep_preproc *without* removing older versions (because scripts/update-tool.py doesn't do that).

Is this what you expect?

Thank you.

**Review by bgruening — APPROVED** [review]
Yes, this looks correct.

**Comment by eschen42** [comment]
> Yes, this looks correct.

Thank you @bgruening

**Pull request merged**